Rem/Utilities/MemeModels/MemeTemplates.cs

```diff
@@ -0,0 +1,37 @@
+﻿using MemeGenerator;
+using System;
+using System.Collections.Generic;
+using SixLabors.ImageSharp;
+using System.Linq;
+
+namespace Rem.Utilities.MemeModels
+{
+    class MemeTemplates
+    {
+        public Dictionary<string, MemeTemplate> Templates { get; set; }
+
+        //globally-shared renderers
```

Comment:
we shouldn't be globally sharing any renderers, the resource usage is not high enough to warrant caching

Reply:
they're 'globally shared' so you can reuse them in the json. It's not related to resource usage.